mgr.go

```diff
@@ -67,21 +67,24 @@ type mountInfo struct {
 }
 
 type containerInfo struct {
-	state          containerState
-	rootfs         string
-	mntPrepRev     []mntPrepRevInfo
-	reqMntInfos    []mountInfo
-	containerMnts  []specs.Mount
-	shiftfsMarks   []configs.ShiftfsMount
-	autoRemove     bool
-	userns         string
-	netns          string
-	netnsInode     uint64
-	uidMappings    []specs.LinuxIDMapping
-	gidMappings    []specs.LinuxIDMapping
-	subidAllocated bool
-	rootfsCloned   bool
-	origRootfs     string // if rootfs was cloned, this is the original rootfs
+	state                  containerState
```

Comment:
> When the container's rootfs is on overlayfs, and Sysbox is using
ID-mapped-mounts on it, then the ID-mapping only applies to the
overlayfs lower layers, but not the upper layer.

Just for curiosity, do you know why is this the case? I would expect id-mapping to be properly handled at upper layers too, sounds like a design bug unless i'm missing something (users shouldn't need to take care of the chown stuff).

Reply:
I don't know why the overlayfs developers decided that only lower layers can be ID-mapped. But I know that I tried ID-mapping the upper layer too and things failed (i.e., mounting overlayfs on the "merged" dir with an ID-mapped upper layer fails). I would have been better to ID-map the upper and lower layers in my view.